etna/commands/backtest_command.py

```diff
@@ -82,13 +82,13 @@ def backtest(
         freq_init = freq
         parse_dates = ["timestamp"]
 
-    df_timeseries = pd.read_csv(target_path, parse_dates=parse_dates)
+    df_timeseries = pd.read_csv(target_path, parse_dates=parse_dates, dtype={"segment": str})
```

Comment:
We should probably add some test that checks this behavior. Smth like en example in the issue.